pkg/util/crypt/crypt_dev_test.go

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2018-2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package crypt
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/sylabs/singularity/internal/pkg/test"
+	"github.com/sylabs/singularity/internal/pkg/util/fs"
+	"github.com/sylabs/singularity/internal/pkg/util/fs/squashfs"
+)
+
+func TestEncrypt(t *testing.T) {
+	test.EnsurePrivilege(t)
+	defer test.ResetPrivilege(t)
+
+	dev := &Device{}
+
+	emptyFile, err := ioutil.TempFile("", "")
```

Comment:
I think it would be good to create these `tempfile`s in a function since its all very common. Can you open a bug for us to go back and review that tech debt after we create a function / test helper for that?

Include a link to the following: https://speakerdeck.com/mitchellh/advanced-testing-with-go?slide=29

Reply:
https://github.com/sylabs/singularity/issues/4071